docs/ibet_wallet_api_v2.yaml

```diff
@@ -3380,6 +3380,9 @@ paths:
                 type: array
                 items:
                   type: string
+              include_canceled_items:
```

Comment:
Additional input fields(optional)
Enter if you wish to return cancelled orders and agreements.